perf/benchmark.js

```diff
@@ -3,11 +3,13 @@
 var _ = require("lodash");
 var Benchmark = require("benchmark");
 var exec = require("child_process").exec;
+var execSync = require("child_process").execSync;
```

Comment:
`execSync` is only node 0.12 and later, right?  I still run these in node 0.10 to compare.